config/locales/es.yml

```
@@ -154,7 +154,7 @@ es:
       title: Dashboard
   dependencies:
     show:
-      click_to_expand:
+      click_to_expand: Pincha en el icono de flecha para expandir
```

```suggestion
      click_to_expand: Haz clic en el icono de flecha para expandir
```
`Clic` is used in other parts of the `es.yml`